[MOB-1933] - AndroidSDK 3.2.5 changes in changelog (#256)
Iterable/iterable-android-sdk: CHANGELOG.md

#### Fixed
- Fixed an issue where in-app click events were not being registered after displaying an in-app message.
- Fixed NullPointerExceptions in `IterableInAppFragmentHTMLNotification` that could occur in some cases when the activity is destroyed and recreated.
- Fixed an issue where in-app messages were not getting cleared upon logout.

> - Fixed an issue where `disableToken` could not disable all devices with same deviceToken

Reviewer: ```suggestion
- `disableToken` now disables all devices with the current deviceToken when `email` and `userId` are not set.
```
+ move this to `Changed`
